src/app/component/statistics/helpers/group/group.component.ts

```diff
@@ -53,8 +53,11 @@ export class GroupComponent implements OnInit {
   constructor() {}
 
   ngOnInit() {
-    const midIndex = Math.ceil(this.statistics.length / 2);
+    if (!this.statistics) {
+      return;
+    }
 
+    const midIndex = Math.ceil(this.statistics.length / 2);
```

Comment:
what's all this about?

seems like this should be the job of css. e.g. a two-column flex layout

Reply:
I had a crack at it, however my best attempt using flexbox wasn't usable.

Reply:
i'd suggest just going to a single column then